JetAnalyzers/bin/jet_match_x.cc

```diff
@@ -0,0 +1,1695 @@
+// ROOT Libraries
```

Comment:
Please add a section to https://github.com/cms-jet/JetMETAnalysis/blob/master/JetAnalyzers/bin/README.md. I'm not sure what this code is suppose to do and how it's different from jet_synchtest_x.cc. If you're renaming the file that is not reflected in this PR.

Reply:
The jet_match_x.cc code is actually a modification of the jet_synchtest_x.cc one, which we do not use anymore. It implements an option to remove a phi slice (for the HEM issue studies). So it's basically a rename, as you said, and I will add that in the description of the code to avoid confusion.

Reply:
But does it really provide fundamentally different functionality? I think it would be better to add a "remove phi-slice" option to jet_synchtest_x.cc, if it's essentially the same thing

Reply:
I agree with you, I will remove the jet_match_x.cc code and re-upload the jet_synchtest_x.cc with the latest updates.